Sources/Classes/Controllers/BaseConfigurableController.swift

```diff
@@ -25,9 +25,39 @@ import UIKit.UIViewController
 /// Base controller that should be configured with view model.
 open class BaseConfigurableController<ViewModel>: UIViewController, ConfigurableController {
 
+    /// Ability to set forced screen orientation
+    open var forcedInterfaceOrientation: UIInterfaceOrientation?
+
     /// A view model instance used by this controller.
     public let viewModel: ViewModel
 
+    override var interfaceOrientation: UIInterfaceOrientation {
+        return forcedInterfaceOrientation ?? super.interfaceOrientation
+    }
+
+    open override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
+        switch interfaceOrientation {
```

Comment:
return super.supportedInterfaceOrientations if forcedInterfaceOrientation is nil